backend/nodes/image_dim_nodes.py

```diff
@@ -27,8 +27,14 @@ def __init__(self):
         )
         self.inputs = [
             ImageInput(),
-            BoundedNumberInput(
-                "Scale Factor (%)", minimum=0.1, maximum=None, default=100.0, step=25.0
+            NumberInput(
+                "Scale Factor",
+                precision=4,
+                step=0.0001,
+                controls_step=25.0,
+                minimum=0,
```

Comment:
Minimum=0 seems to be very common. Could we make this the default?

Reply:
Yeah, that's fair.